src/Perf/IIDOptimizer/SignatureGenerator.cs

```diff
@@ -170,6 +174,13 @@ public SignaturePart GetSignatureParts(TypeReference type)
                 return new RuntimeClassSignature(type, GetSignatureParts(iface));
             }
 
+            // For types projected from WUX or MUX into .NET, we'll need to do a runtime lookup for the IID.
+            // TODO-WuxMux: We can instead take an option in the IID optimizer to hard-code the lookup for WUX or MUX when specified, which would be more efficient for scenarios where this is possible.

```

Comment:
I think we can do it, but since it's not needed for the MVP of this feature, I'd rather not fight MSBuild and the command line to get it working.

I don't remember if we run the IID optimizer on library assemblies too though, and I think you could build a library that works with both WUX and MUX if you're writing it in C# and it only uses the mapped WUX/MUX types.